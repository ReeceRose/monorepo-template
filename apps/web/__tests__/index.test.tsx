import { render } from '@testing-library/react';

import Home from '@/pages/index';

describe('Index page tests', () => {
  test('Index page matches snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
