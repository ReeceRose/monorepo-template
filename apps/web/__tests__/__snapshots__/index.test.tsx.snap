// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page tests Index page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Web
    </h1>
    <button
      class="text-xl"
    >
      Boop
    </button>
  </div>
</DocumentFragment>
`;
